tool/pom.xml

```diff
@@ -82,6 +82,7 @@
                                         <mainClass>com.fidesmo.fdsm.Main</mainClass>
                                     </classPath>
                                     <jre>
+                                        <path>${env.JAVA_HOME}</path>
```

Comment:
How bad is it if the user has not defined a `JAVA_HOME`? I ask because I'm not certain that all users will have that env var set. Maybe we can add something in the README explaining that `JAVA_HOME` is mandatory?

Reply:
Maven doesn't work without `JAVA_HOME` set at all. Also this specific goal is only run during the release phase which is a GH Action only and not intended for a local run, so don't think it needs additional documentation

Reply:
Ah ok, then it's good to merge.